exporter/sapmexporter/config.go

```diff
@@ -61,33 +60,29 @@ type Config struct {
 	exporterhelper.RetrySettings   `mapstructure:"retry_on_failure"`
 }
 
-func (c *Config) validate() error {
+func (c *Config) Validate() error {
 	if c.Endpoint == "" {
 		return errors.New("`endpoint` not specified")
 	}
-
-	e, err := url.Parse(c.Endpoint)
+	_, err := url.Parse(c.Endpoint)
 	if err != nil {
 		return err
 	}
-
-	if e.Scheme == "" {
-		e.Scheme = defaultEndpointScheme
-	}
-	c.Endpoint = e.String()
-	return nil
-}
-
-func (c *Config) Validate() error {
 	if err := c.QueueSettings.Validate(); err != nil {
 		return fmt.Errorf("sending_queue settings has invalid configuration: %w", err)
 	}
```

Comment:
AFAIR this is also not needed anymore, but please verify before removing

Reply:
if that's true I can delete even more code ;)

Reply:
Confirmed. I have a test showing the component.ValidateConfig method will test queue settings.